Softeq.XToolkit.Permissions/IPermissionsManager.cs

```diff
@@ -6,10 +6,30 @@
 
 namespace Softeq.XToolkit.Permissions
 {
+    /// <summary>
+    ///     Main interface for check/request permissions, for use in your logic.
+    /// </summary>
     public interface IPermissionsManager
     {
+        /// <summary>
+        ///     Determines whether this permission has been granted.
+        ///     Otherwise, uses dialog for the pre-notify user and then make requests permissions or opens Settings if needed.
```

Comment:
This sentence is not clear as well :)
Sounds like it should be "for pre-notifying the user and then making permission request or opening Settings if needed"